patches/chrome-android-chrome_public_apk_tmpl.gni.patch

```diff
@@ -1,16 +1,17 @@
 diff --git a/chrome/android/chrome_public_apk_tmpl.gni b/chrome/android/chrome_public_apk_tmpl.gni
-index 4438f47bcfdb8cbbc1428b35d22bff3aa3f484c4..5b4a19980bb79d750922b61e75faad5afe654e2e 100644
+index f5fab3e7ff209d13bddf0e0107a5318cc4b97eca..8018680d5b5db8fc96219affc33dc80129814561 100644
 --- a/chrome/android/chrome_public_apk_tmpl.gni
 +++ b/chrome/android/chrome_public_apk_tmpl.gni
-@@ -227,6 +227,7 @@ template("chrome_public_common_apk_or_module_tmpl") {
+@@ -227,6 +227,8 @@ template("chrome_public_common_apk_or_module_tmpl") {
        "${_material_package}:[Ss]lider",
        "${_material_package}:[Tt]oolbarLayout",
      ]
 +    resource_exclusion_exceptions += brave_resources_exclusion_exceptions
-     resource_values_filter_rules -= ["${_material_package}:[Ff]loatingActionButton"]
++    resource_values_filter_rules -= ["${_material_package}:[Ff]loatingActionButton"]
```

Comment:
this can be on the same line as `resource_exclusion_exceptions`

Reply:
I see you did this here https://github.com/brave/brave-core/pull/6042/commits/8ec068061445ce93563f76f31f03be6466649ef2